server/py/framework/db/sqldb/models.py

```diff
@@ -200,9 +200,7 @@ class Notification(Base, mlrun.utils.db.BaseModel):
     warnings.simplefilter("ignore")
 
     # deprecated, use ArtifactV2 instead
-    # TODO: remove in 1.8.0. Note that removing it will require upgrading mlrun in at least 2 steps:
-    #  1. upgrade to 1.6.x which will create the new table
-    #  2. upgrade to 1.7.x which will remove the old table
```

Comment:
The versions in the comment are not correct, but the 2-phase procedure will still be required once we remove the table.

Reply:
I think it is reflected in the new comment.
Once data migration v5 is removed,  we can assume mlrun >= 1.6 (i think 1.6) pre upgrade. Then it will also be safe to remove the table in the same version.